orc8r/cloud/go/services/eventd/eventd/main.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"magma/gateway/eventd"
+	"magma/orc8r/cloud/go/obsidian/swagger"
+	swagger_protos "magma/orc8r/cloud/go/obsidian/swagger/protos"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/cloud/go/service"
+
+	"github.com/golang/glog"
+)
+
+func main() {
+	// Create the service
+	srv, err := service.NewOrchestratorService(orc8r.ModuleName, eventd.ServiceName)
+	if err != nil {
+		glog.Fatalf("Error creating service: %s", err)
```

Comment:
Use `%+v` for logging errors rather than `%s` -- the former includes stack trace, latter just includes the error string